tests/http-api.spec.ts

```diff
@@ -254,11 +284,11 @@ describe('http api', function () {
       expect(body.error).to.not.exist;
 
       const { reply } = body.result;
+      console.log(reply);
```

Comment:
Are we logging this here on purpose during testing? If necessary, should we add some label?

Reply:
No, must've forgotten to remove after some sanity testing.

Removed.